Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN \
     cd app && go build -o /build/reproxy -ldflags "-X main.revision=${version} -s -w"
 
 
-FROM ghcr.io/umputun/baseimage/app:v1.12.0 as base
+FROM ghcr.io/umputun/baseimage/app:latest as base
```

Comment:
I’d rather have the tagged version here instead of the latest. You know how sometimes incompatible changes sneak in and cause unexpected results? Well, for something as fundamental as this, it’s worth the extra effort to maintain compatibility.

Reply:
In this case it makes no sense as we only copying certificate and timezone files from the base image, but I pinned it to v1.15.0 instead in the new version of the commit.